wolfssl/wolfcrypt/settings.h

```diff
@@ -1087,7 +1087,8 @@ extern void uITRON4_free(void *p) ;
              *     heap_caps_realloc(p, s, MALLOC_CAP_8BIT)
              * There's no pvPortRealloc available:  */
             #define XREALLOC(p, n, h, t) ((void)(h), (void)(t), realloc((p), (n)))
-        #elif defined(USE_INTEGER_HEAP_MATH) || defined(OPENSSL_EXTRA)
+        #elif defined(USE_INTEGER_HEAP_MATH) || defined(OPENSSL_EXTRA) || \
```

Comment:
Is this because the OPENSSL_ALL doesn't force on OPENSSL_EXTRA until later in settings.h? I recall we had added some logic to force OPENSSL_EXTRA to get set with OPENSSL_ALL.

Reply:
yes, OPENSSL_EXTRA gets defined around line 2973 if OPENSSL_ALL is defined